tests/test_service_cli.py

```diff
@@ -56,21 +56,20 @@ def test_version_action(option):
     assert __version__ in result.output
 
 
-@pytest.mark.parametrize("sig", [signal.SIGINT, signal.SIGTERM])
+@pytest.mark.parametrize(["sig", 'patch_logger'], [[signal.SIGINT, False], [signal.SIGTERM, False]], indirect=["patch_logger"])
```

Comment:
Just curious, what does `indirect` do here?

Reply:
TIL about it! It's how you pass arguments to pytest fixtures. So this is saying, "For the parametrized `patch_logger` values, don't actually pass them directly as parameters, but as parameters of that parameter". Check out: https://docs.pytest.org/en/latest/example/parametrize.html#apply-indirect-on-particular-arguments